packages/web-components/fast-foundation/src/button/button.ts

```diff
@@ -180,9 +180,19 @@ export class Button extends FormAssociatedButton {
     };
 
     /**
+     * This has been deprecated in favor of control property
      * References the root element
+     * @deprecated This has been deprecated in favor of control property
```

Comment:
This explanation might be confusing because `control` isn't public and documented. Probably okay to just omit the message.
```suggestion
     * @deprecated
```

Reply:
Unfortunately this throws an error in the build if it doesn't include the message. As internal it may not...I'm open to try it, but we may need to add it back in